### PR TITLE
test: added test to udp_client

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

The functions of the udp_client module were tested to test if the emUnit library was used.

In this PR, during the tests, the udp server module presented some errors, these were:

1) Wrong name of include folder inside upd_server
2) buffer overflow when callback was executed

Also added `test_utils.config` and `test_print_stack_usage.config` since when we ran make test this would throw an error.

### Testing procedure

cd test/udp_client

make BOARD="your_board" flash test

### CODE REVIEW

**Comment by luisan00** [comment]
:clap: request :clap: review :clap: only :clap: when :clap: all :clap: test :clap: are :clap: green

**Review by luisan00 — CHANGES_REQUESTED** [review]
:clap: request :clap: review :clap: only :clap: when :clap: all :clap: test :clap: are :clap: green

**Comment by eduazocar** [comment]
Please change description to english

File: tests/udp_client/main.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief       storage file
```
Comment (eduazocar) [inline tests/udp_client/main.c:18]: Change reference to Test_udp_client

**Review by eduazocar — COMMENTED** [review]
Everything looks good for me, do squash to approve this

**Review by eduazocar — APPROVED** [review]
Go ahead :+1:

File: tests/udp_client/main.c [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ */
+#include <string.h>
+#include <errno.h>
+#include <stdio.h>
+
+#include "embUnit.h"
+#include "udp_client.h"
+#include "udp_server.h"
+
+#include "shell.h"
```
Comment (RocioRojas) [inline tests/udp_client/main.c:30]: remove includes: shell, shell_commands, msg, xtimer, board

File: tests/udp_client/doc.txt [hunk 3]
```diff
@@ -9,10 +9,15 @@ For now you can only send messages to ipv6 address.
 
 To test the code from python:
 
+```c
 make flash test
 
+```
+
 to test only C code"
```
Comment (RocioRojas) [inline tests/udp_client/doc.txt:17]: to test only C code:

File: tests/udp_client/main.c [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+#include <errno.h>
+#include <stdio.h>
+
+#include "embUnit.h"
+#include "udp_client.h"
+#include "udp_server.h"
+
+#include "xtimer.h"
+
+
```
Comment (eduazocar) [inline tests/udp_client/main.c:32]: What do you need the xtimer library?

Comment (eduazocar) [inline tests/udp_client/main.c:32]: You have two consecutive empty lines

File: tests/udp_client/main.c (hunk 2 above)
Comment (RocioRojas) [inline tests/udp_client/main.c]: Removed incluedes: shell, shell_commands, msg, xtimer, board

File: tests/udp_client/main.c (hunk 4 above)
Comment (xkevin190) [reply tests/udp_client/main.c]: i remove it

Comment (xkevin190) [reply tests/udp_client/main.c]: removed

**Review by eduazocar — COMMENTED** [review]
Good, do squash to approve this

**Review by eduazocar — APPROVED** [review]
Good :+1: This is approved

**Review by luisan00 — APPROVED** [review]
👍

**Pull request merged**